packages/augur-state/src/db/AbstractDB.ts

```diff
@@ -14,9 +14,13 @@ export abstract class AbstractDB {
   protected db: PouchDB.Database;
   public readonly dbName: string;
 
-  constructor (dbName: string) {
+  protected constructor (dbName: string) {
```

Comment:
My IDE recommended this. I assume this prevents the abstract class from being instantiated but I'd also assume that classifying it as "abstract" did that.